spk/borgbackup/Makefile

```diff
@@ -27,6 +27,12 @@ HOMEPAGE = https://borgbackup.readthedocs.io
 LICENSE  = BSD-3-Clause
 
 SERVICE_SETUP = src/service-setup.sh
+
+include ../../mk/spksrc.common.mk
+ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
```

Comment:
Since `usr-local-linker` was introduced since `6.0-5941` why not use for the DSM6 as well?

Reply:
Yes, lets do it in another PR.